lib/jsonld-signatures.js

```diff
@@ -228,24 +264,61 @@ api.verify = util.callbackify(async function(input, options) {
   if(options.documentLoader) {
     opts.documentLoader = options.documentLoader;
   }
+  // Why expand just to frame again?  The answer is in the proofPurpose
```

Comment:
Should these comments be left in?

Reply:
They seem like they are trying to be helpful but aren't quite there. So I'd like to keep them in, but have them actually say what "the answer" is rather than guessing what the author's intent was. For example:

```Here we expand and then frame again because ...```

Also... there's a flag to skip expansion when framing and doing other ops if the input is known to already be expanded. I didn't look to see if we're doing that here ... just responding to the comment that only exposed so much contextual code.

Reply:
@dlongley a couple points of clarification.  The statements made in the comments are accurate wrt the necessity of working with expanded data?

Second, the prose and variable names are referring to `framed` but the operation being referred to is `jsonld.compact` (which is btw using the skipExansion flag).

https://github.com/digitalbazaar/jsonld-signatures/blob/4d8384ceac5082da0b0c2cb432ddcaca295ac23a/lib/jsonld-signatures.js#L267-L279

Reply:
@mattcollier, I'm pretty sure @cwebber wrote those statements... @cwebber can you weigh in on what's going on here? @gannan08 you may also have comments.

Reply:
@dlongley upon closer inspection the use of the variable `framed` for a `jsonld.compact` operation predates this PR: https://github.com/digitalbazaar/jsonld-signatures/pull/37/files#diff-a1df416cb75df6428c21b0be5fd80886R275

I'm just expecting that `framed` would be associated with a `frame` operation.  Maybe this is not an accurate expectation.

Reply:
@mattcollier @dlongley We no longer need to pass the expanded input into proofPurpose handlers. Documents passed into the proofPurpose handlers and documents that are outputted are expected to be compacted using the security context. Expansion should be handled in the proofPurpose libs when necessary.